charger/amtron.go

```diff
@@ -91,18 +91,17 @@ func (wb *Amtron) Status() (api.ChargeStatus, error) {
 		return api.StatusNone, err
 	}
 
-	switch b[0] {
+	switch b[1] {
 	case 1, 2:
 		return api.StatusA, nil
-	case 3, 4:
+	case 3, 4, 5:
```

Comment:
5 (C1) ist schon C nicht B. Das ist der normale Ladezustand.

```suggestion
	case 3, 4:
```